lib/galaxy/datatypes/text.py

```diff
@@ -369,8 +392,8 @@ class SnpSiftDbNSFP( Text ):
     """
     def __init__( self, **kwd ):
         Text.__init__( self, **kwd )
-        self.add_composite_file( '%s.grp', description='Group File', substitute_name_with_metadata='reference_name', is_binary=False )
-        self.add_composite_file( '%s.ti', description='', substitute_name_with_metadata='reference_name', is_binary=False )
+        self.add_composite_file( '%s.gz', description='dbNSFP bgzip', substitute_name_with_metadata='reference_name', is_binary=True )
+        self.add_composite_file( '%s.gz.tbi', description='Tabix Index File', substitute_name_with_metadata='reference_name', is_binary=True )
```

Comment:
@jmchilton   Probably not.  But that might be a good thing because that likely didn't work.  I've checked this out with the composite dataloader.